CHANGELOG.md

```diff
@@ -7,7 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-## [0.33.0] - 2024-10-09
+### Added
+
+- `BaseEventListener.flush_events()` to flush events from an Event Listener.
+
+### Changed
+
+- **BREAKING**: `BaseEventListener.publish_event` `flush` argument. Use `BaseEventListener.flush_events()` instead.
+
+### Fixed
+
+- Structures not flushing events when not listening for `FinishStructureRunEvent`.
+
+## \[0.33.0\] - 2024-10-09
```

Comment:
weird escape characters?

Reply:
Markdown formatter does it. This was already present in the repo wide format.